src/core/lib.lua

```diff
@@ -5,6 +5,8 @@ local C = ffi.C
 local getopt = require("lib.lua.alt_getopt")
 local syscall = require("syscall")
 require("core.clib_h")
+local band, bor, bnot, lshift, rshift, bswap =
```

Comment:
Since you're already doing this change, I think it would be also convenient to add an explicit inclusion of the BitOp module. It's not necessary, but the docs recommend it (and other modules do it too):

_The suggested way to use the BitOp module is to add the following to the start of every Lua file that needs one of its functions [...]This makes the dependency explicit, limits the scope to the current file and provides faster access to the bit.* functions, too._